uiowa/uiowa
# remove 'menuparent' class from cloned parent link to prevent FontAwes…

Remove 'menuparent' class from cloned parent link to prevent FontAwesome expand icon from being shown. fixes #3902



# How to test

On a mobile device, check to make sure the cloned parent link doesn't have a missing FontAwesome icon showing (usually an [X]-like icon)
